app/views/fsss_view.py

```diff
@@ -54,6 +54,8 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
   preferred_name = get_emp_info('preferredName', on_false='None')
 
+  employee_info['address'] = map_full_address_details(employee_info)
```

Comment:
Mutating one of the inputs in this way is a solution that's prone to error. Unless you check the entire function, you cannot know that `employee_info` was tampered with in this way.

A more reliable system would be to take `'address'` out of the `EMP_CONTACT_FIELDS` dictionary, and manually add the address to `data_contact = {}`, by changing it to `data_contact = {'Address': map_full_address_details(employee_info)}`

Reply:
Have changed this in all the views